.buildkite/scripts/backport_branch.sh

```diff
@@ -0,0 +1,149 @@
+  if [[ $responseCode == "200" ]]; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+isCommitExist() {
+  local commit_sha=$1
+  local branch=$2
+  git checkout $branch
+  local searchResult="$(git branch --contains $commit_sha | grep $branch | awk '{print $2}')"
+  echo "${searchResult}"
+  git checkout $BUILDKITE_BRANCH
+  if [ "${searchResult}" == "${branch}" ]; then
+    echo "The commit $commit_sha exists in the branch $branch"
+    return 0
+  else
+    echo "The commit $commit_sha doesn't exist in the branch $branch"
+    return 1
+  fi
+}
+
+isBranchExist() {
+  local branch=$1
+  local searchResult="$(git branch | grep $branch | awk '{print $2}')"
+  echo "${searchResult}"
+  if [ "${searchResult}" == "${branch}" ]; then
+    echo "The backport branch $branch has already exist"
+    return 0
+  else
+    echo "The backport branch $branch hasn't exist"
+    return 1
+  fi
+}
+
+createLocalBackportBranch() {
+  local branch_name=$1
+  local source_commit=$2
+  if git checkout -b $branch_name $source_commit; then
+    echo "The branch $branch_name has created."
+  else
+    buildkite-agent annotate "The backport branch **$BACKPORT_BRANCH_NAME** hasn't created." --style "warning"
+    exit 1
+  fi
+}
+
+removeAllPackages() {
+  for dir in "$PACKAGES_FOLDER_PATH"/*; do
+    if [[ -d "$dir" ]] && [[ "$(basename "$dir")" != "$PACKAGE_NAME" ]]; then
+      echo "Removing directory: $dir"
+      rm -rf "$dir"
+    fi
+  done
+}
+
+processFifes() {
+  local BUILDKITE_FOLDER_PATH=".buildkite"
+  local JENKINS_FOLDER_PATH=".ci"
+  git checkout $BACKPORT_BRANCH_NAME
+  ls -la                                                                        #TODO remove after tests
+  echo "Copying $BUILDKITE_FOLDER_PATH..."
+  git checkout $SOURCE_BRANCH -- $BUILDKITE_FOLDER_PATH
+  echo "Copying $JENKINS_FOLDER_PATH..."
+  git checkout $SOURCE_BRANCH -- $JENKINS_FOLDER_PATH
+  ls -la                                                                        #TODO remove after tests
+  ls -la $BUILDKITE_FOLDER_PATH                                                 #TODO remove after tests
+  ls -la $JENKINS_FOLDER_PATH                                                   #TODO remove after tests
+
+  if [ "${REMOVE_ALL_PACKAGES}" == "true" ]; then
+    echo "Removing all packages from $PACKAGES_FOLDER_PATH folder"
+    removeAllPackages
+    ls -la $PACKAGES_FOLDER_PATH
+  fi
+
+  echo "Commiting and pushing..."
+  # git add $BUILDKITE_FOLDER_PATH
+  # git add $JENKINS_FOLDER_PATH
+  # git commit -m "Add $BUILDKITE_FOLDER_PATH and $JENKINSFILE_PATH to backport branch: $BACKPORT_BRANCH_NAME from the $SOURCE_BRANCH branch"
+  # git push origin $BACKPORT_BRANCH_NAME
+}
+
+echo "Check if the package has published"
+if ! isPackagePublished "${FULL_ZIP_PACKAGE_NAME}"; then
+  buildkite-agent annotate "The package version: **$FULL_PACKAGE_NAME** hasn't published yet." --style "warning"
+  exit 1
+fi
+
+echo "Check if commit exists."
+if ! -z "$BASE_COMMIT"; then
```

Comment:
There is an error here in the output:
https://buildkite.com/elastic/integrations-backport/builds/15#018c6d8c-9091-462c-a643-a03e140a6436/71-75

I think it's missing square brackets: 
```suggestion
if [ ! -z "$BASE_COMMIT" ]; then
```

Reply:
fixed.